packages/now-cli/src/types.ts

```diff
@@ -205,19 +205,46 @@ type SRVRecordData = {
   };
 };
 
-type MXRecordData = {
-  name: string;
-  type: 'MX';
-  value: string;
+export interface MXRecordData extends GenericDNSRecordData {
   mxPriority: number;
```

Comment:
You need to have `type: 'MX';` here

Reply:
Only now I saw this comment after fixing it. It took me a few minutes...
Thanks @styfle !